mixer_paths.xml

```diff
@@ -285,7 +285,7 @@
         <ctl name="SLIM RX1 MUX" value="AIF1_PB" />
         <ctl name="SLIM_0_RX Channels" value="One" />
         <ctl name="RX7 MIX1 INP1" value="RX1" />
-        <ctl name="RX7 Digital Volume" value="88" />
+        <ctl name="RX7 Digital Volume" value="90" />
```

Comment:
Is this speakerphone volume? I am not sure that change by 2 units only will make much difference?

Reply:
Ok this change was already made on 5.1, since the volume on nexus 5 was too low

Reply:
Above the 90 the sound makes cracking noise